Gemfile

```diff
@@ -3,3 +3,4 @@ source 'https://rubygems.org'
 gem 'watir', '~> 6.19', '>= 6.19.1'
 gem 'webdrivers', '~> 4.6'
 gem 'nokogiri', '~> 1.11', '>= 1.11.7'
+gem 'geckodriver-helper', '~> 0.0.3'
```

Comment:
@raselahmed2k7 san, the [repo](https://github.com/DevicoSolutions/geckodriver-helper) of the gem has only 22 stars on github? Is it safe to use?

Reply:
@antorkhan san, Thank you for the review. Basically geckodriver is used to use Firefox web driver for selenium to set PATH variable instead of doing manually. Hence it is widely used the geckodriver for selenium, sample link https://github.com/mozilla/geckodriver/security. 
Here I may need to more work around if any better similar gem available. But it seems the geckodriver is needed and widely used in selenium.
Thanks